lib/block/block.go

```diff
@@ -23,10 +23,10 @@ type Block struct {
 	ProposerTransaction string   `json:"proposer_transaction"` /* ProposerTransaction */
 	//PrevConsensusResult ConsensusResult
 
-	Hash      string `json:"hash"`
-	Confirmed string `json:"confirmed"`
-	Proposer  string `json:"proposer"` /* Node.Address() */
-	Round     uint64 `json:"round"`
+	Hash          string `json:"hash"`
+	Proposer      string `json:"proposer"` /* Node.Address() */
+	Round         uint64 `json:"round"`
+	ConfirmedTime string `json:"confirmed-time" rlp:"-"`
```

Comment:
@kfangw We have used `Confirmed` term, without `Time` suffix. I wish we follow the traditions. :)

Reply:
@kfangw ~~Is there any reason to use `string` instead of `time.Time`? `time.Time` also can be stored in leveldb. Parsing time string will be cost.~~ I understand to use `string`, it is used for creating key.